spec.html

```diff
@@ -71,7 +71,7 @@ <h1>String.prototype.replaceAll ( _searchValue_, _replaceValue_ )</h1>
           1. Let replacement be GetSubstitution(_searchString_, _string_, _position_, _captures_, *undefined*, _replaceValue_).
       1. Let _stringSlice_ be the substring of _string_ consisting of the code units from _lastPosition_ (inclusive) up through _position_ (exclusive).
       1. Let _result_ be the string-concatenation of _result_, _stringSlice_, and _replacement_.
-      1. Let _lastPosition_ be _position_.
+      1. Let _lastPosition_ be the sum of _position_ and _searchLength_.
```

Comment:
Good catch, thanks! Two comments:

1. Just `Let _lastPosition_ be _position_ + _searchLength_` (instead of 'the sum of')?
2. The name `lastPosition` may be a bit confusing if it doesn't actually store the previous value of `position`. I don't have a great idea for another name though, perhaps something like `endOfLastMatch`?

Reply:
Addressed